Monika After Story/game/script-affection.rpy

```diff
@@ -170,30 +180,26 @@ init python:
     #Internal cooldown to avoid topic spam and Monika affection swings, the amount of time to wait before a function is effective
     #is equal to the amount of points it's added or removed in minutes.
 
-    #Monika's initial affection based on start-up. Need to decide on super exp before we do anything else...
-    #Monika closed game herself and how happy she is determines on time between closed game and reopening.
-    #if persistent.closed_self == True:
-       # if datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 6):
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 6) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 12):
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 12) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(hours = 18):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(hours = 18) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 1):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 1) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 2):)
-
-
-       # elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 2) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 4):
-
-
-        #elif datetime.datetime.now() > persistent.sessions["last_session_end"] + datetime.timedelta(days = 4) and datetime.datetime.now() < persistent.sessions["last_session_end"] + datetime.timedelta(days = 14):
-
-
+    #Monika's initial affection based on start-up.
+    if persistent._mas_long_absence == False:
+        if persistent.sessions["last_session_end"] == None:
```

Comment:
checking for `None` should be `if x is None`

Although in this case, since you just `pass` through the block, it'd be better to make the condition `if persistent.sessions["last_session_end"] is not None:` and get rid of the `else`:
```python
if x is not None:
    time_difference .....
```